frontend/src/components/ChangePasswordDialog.js

```diff
@@ -37,64 +45,63 @@ const ChangePasswordDialog = ({ dialogOpen, handleCloseDialog }) => {
       setNewPassword('')
       setNewPasswordTextFieldChanged(false)
       handleCloseDialog()
+      handleChangePasswordSuccessOpenAlert()
     } catch (err) {
       console.log('error: ', err)
     }
   }
 
+  const renderChangePasswordSuccessAlert = () => {
+    if (!changePasswordSuccessAlertOpen) {
+      return null
+    }
+
+    return (
+      <SnackbarAlert
+        alertOpen={changePasswordSuccessAlertOpen}
+        setAlertOpen={setChangePasswordSuccessAlertOpen}
+        severity="success"
+        alertMsg="Your password has been changed successfully!"
+      />
+    )
+  }
+
   return (
-    <Dialog fullWidth={true} maxWidth="sm" open={dialogOpen}>
-      <DialogTitle>Change Password</DialogTitle>
-      <DialogContent dividers>
-        <DialogContentText>
-          Please enter your new password below and submit.
-        </DialogContentText>
-        {!newPassword && !newPasswordTextFieldChanged && (
+    <>
+      <Dialog fullWidth={true} maxWidth="sm" open={dialogOpen}>
+        <DialogTitle>Change Password</DialogTitle>
+        <DialogContent dividers>
+          <DialogContentText>
+            Please enter your new password below and submit.
+          </DialogContentText>
           <TextField
             autoFocus
+            error={!newPassword && newPasswordTextFieldChanged}
```

Comment:
Actually could I ask what is the usage of `newPasswordTextFieldChanged`? It seems like it is not used independently and always used with `newPassword`, so is it necessary?

Reply:
The usage of `newPasswordTextFieldChanged` is to check whether the `TextField` has been changed the first time (starting value of newPasswordTextFieldChanged is false), then it will be set to true (line 93).